test/global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We now have 3 global-jsons. 

We probably don't need them all.

Reply:
The global.jsons under src and test are for building with preview version.
About the third one under repos root folder: there is a recently added msbuild task in yaml, named "Ensure msbuild.exe can parse nuget.sln". It would fail for if there is no global.json under repos root folder. (Failed build is here https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=2811147)
Shall I just keep the one under root folder?

Reply:
Yeah, given that we have the one in the root, I don't think we need one in test/src dirs. 
Unless you can think of a reason :)